RetailCoder.VBE/UI/HotKeyDisplayConverter.cs

```diff
@@ -36,7 +36,7 @@ public static string Convert(string value)
         public static string ConvertBack(string value)
         {
             var tuple = _keys.SingleOrDefault(k => k.Item2 == value);
-            return tuple == null ? value : tuple.Item1.ToString();
+            return tuple?.Item1.ToString() ?? value;
```

Comment:
similar concerns as with the previous refactoring involving null-coalescing. Item1 can be null this way....

Reply:
If `tuple?.Item1` is `null`, the whole expression stops.